aries-site/src/examples/components/pageheader/PageHeaderContentRegions.js

```diff
@@ -33,17 +33,15 @@ export const PageHeaderContentRegions = ({ background, pad, ...rest }) => {
           border={{ style: 'dashed' }}
           round="xxsmall"
         >
-          <Heading size="small" margin="none">
-            L2Pod-FTC02 Device
-          </Heading>
+          <Heading margin="none">L2Pod-FTC02</Heading>
```

Comment:
Good question! `1` is the [default level](https://v2.grommet.io/heading#level). In this case and some of your other comments, since it is the page title then level 1 is appropriate and we don't need to add a level prop.

Reply:
Thanks for clarifying!